lookout/style/format/features.py

```diff
@@ -273,6 +273,10 @@ def extract_features(self, files: Iterable[File], lines: List[List[int]]=None
             labels.append([vnode.y for vnode in vnodes if vnode.y is not None and
                            (vnode.start.line in file_lines if file_lines is not None else True)])
 
+        if not labels:
+            # nothing was extracted
+            return None, None, None
```

Comment:
I think it might be better to return only one `None` and have the type changed to `Optional[Tuple[…]]`. We'll have to check for `None`-s now and won't ever check on the 3 values independently so it might be more consistent with our usage.

Reply:
yes, it's a good option because tuple of 3 None's is not the most obvious thing you expect to be returned.